skupstina_django/scraper/db_utils.py

```diff
@@ -0,0 +1,80 @@
+import dateparser
```

Comment:
Cool, I didn't know about this package.

Reply:
Yeah! Very cool package for date parsing in general and for dates in foreign languages.